### PR TITLE
Update certificate generation script

### PR DESCRIPTION
**Pull request by rap-spyro**
Provide an updated version of the certificate generation script that enables creating device certificate separately. 
Additionally provide an option to set the certificate password instead of hardcoded value.

### CODE REVIEW

File: scargo/certs/certGen.sh [hunk 1]
```diff
@@ -19,8 +19,10 @@ set -o pipefail # Exit if pipe failed
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ACTION_COMMAND=""
 ACTION_PARAM=""
-DEVICE_NAME="new-device"
-OUTPUT_DIR=/repo/out/cert
+ACTION_PARAM2=""
+DEVICE_NAME="iot-device"
+OUTPUT_DIR=/workspace/build/cert
```
Comment (PUR-7176) [inline scargo/certs/certGen.sh:24]: it will be better to have /workspace/ as a  param i.e. - scargo project dir where gen command was trigger

**Review by PUR-7176 — COMMENTED** [review]
presently ca.pub is also copy from scargo (the old Baltimore). Most likely need to be updated of new ca.pub

**Comment by rap-spyro** [comment]
> presently ca.pub is also copy from scargo (the old Baltimore). Most likely need to be updated of new ca.pub

Do you mean `ca.pem`? It's downloaded if not already present [here](https://github.com/Spyro-Soft/scargo/pull/104/files#diff-8abe3e037c0eb4aa9c7de9a64fa022b21a54fde71adb1a469acd28a93dd9696dR99) (renamed later). Should the tool do something else?

**Comment by PUR-7176** [comment]
> presently ca.pub is also copy from scargo (the old Baltimore). Most likely need to be updated of new ca.pub
out of the scope

**Review by PUR-7176 — APPROVED** [review]

**Review by aaksenczuk — APPROVED** [review]

**Pull request merged**